### PR TITLE
Disable sound option in settings

### PR DESCRIPTION
**Pull request by avatsaev**

### CODE REVIEW

File: src/renderer/lib/prefs.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
```
Comment (dcposch) [inline src/renderer/lib/prefs.js]: 99% sure we don't need this file

The state object already contains prefs

**Comment by dcposch** [comment]
Otherwise, looks reasonable to me

@feross, what do you think? Maybe we can just make the sound effects a bit more subtle. Adding a checkbox to Preferences just for disabling sound effects doesn't feel very minimalist

**Comment by avatsaev** [comment]
To be honest with you, from UX stand point, 99% of the sound effects in the app are not necessary, usually sounds are used to make user pay attention to an important event, like a notification for example, but start up sound? Play sound? They are more of an annoyance than convinience

**Comment by feross** [comment]
-1 from me. Let's just make the right decision with regards to sounds. Adding a preference is a cop-out IMO.

**Comment by avatsaev** [comment]
The right decision would be to remove them all together

**Comment by feross** [comment]
You're the first person to express this opinion, so I disagree that it's clearly the right decision. Thanks for sharing your thoughts nonetheless.

**Comment by avatsaev** [comment]
https://github.com/feross/webtorrent-desktop/issues/858

**Comment by Grogs** [comment]
+1 for no sounds.

**Comment by feross** [comment]
I just sent a PR to make the UI sounds subtler. See https://github.com/feross/webtorrent-desktop/pull/945 I hope you give it a try by cloning `master` or waiting until the next release comes out. Would be great to get your feedback.

I'm going to close this PR now.

**Pull request closed without merging**